### ISSUE TITLE
Sign up Now Text & Title do not match

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#108]
This is an accessibility concern for low vision users who use Screen reader + magnifier. When the user hears Subcribe to TALA news letters, he/she will be looking for the same text for the button. We have to be consistent with the visual name & screen reader title.

**NVDA speech viewer**
![image](https://cloud.githubusercontent.com/assets/15111528/10991016/9409619c-84ad-11e5-9ed9-c2f9529986b2.png)

![image](https://cloud.githubusercontent.com/assets/15111528/10991019/a090a0ce-84ad-11e5-8874-8b0f0c6c896e.png)

**Comment by HerinHentry** [comment]
As per judging form
If the image consists of text, the alt-text should convey the same information and the alt text should be the same or clearer.  Each active image without appropriate alt text or lacking an alt attribute is an error.

**Comment by mr-bobz** [comment]
It is not an image. Anyway, should be fine now.

**Comment by HerinHentry** [comment]
<...a title="Sign Up Now (opens new window to 3rd party site)" class="btn btn-custom-black" aria-label="Opens New Window" href="http://visitor.r20.constantcontact.com/d.jsp?llr=67lifhpab&amp;p=oi&amp;m=1115858153373&amp;sit=lxc8balib&amp;f=b119c4d1-f7ba-41c3-a67b-2afc56cca4fd" target="_blank">Sign Up Now<.../a>

Excellent. Can be closed.
